ui/shared/PushHealthStatus.jsx

```diff
@@ -92,13 +92,12 @@ class PushHealthStatus extends Component {
     }
 
     return (
-      <React.Fragment>
+      <span data-testid={`health-status-${revision}`}>
```

Comment:
This means, in the tests, I can check for its existence before having to wait for the API call to come back.